package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "grace-shopper-funky-monkey",
-  "version": "0.3.0",
+  "version": "0.5.0",
```

Going to merge Erin's 2 PR's first, your version should change.

```suggestion
  "version": "0.7.0",
```